### ISSUE TITLE
Offset retrieving method

### ISSUE THREAD

**Issue by DjTrilogic** [l4v4ph0x/floral_flyff_cheat#1]
Hey LavaPhox,

First of all, thank's for this amazing work !

It would be great if you share your method of finding offsets so that forks can be adapted to other P-Servers.

Do you think that can be possible ?

Thank's again !

**Comment by DjTrilogic** [comment]
Hey again,

I was able to find all the offsets using Cheat Engine (I just learned some tips); Actually, finding offsets is pretty easy 👍 (The bot works like a charm now)

Thank's a lot for commenting your code, It facilitated a lot my searches.

I will try to learn some Lua to automate offset searches. Once I do that, i'll open a PR.

**Comment by DjTrilogic** [comment]
To be honest your code is pretty understandable & well commented. However, I think there are many possible optimizations.

What do you think about a code refactoring ?

**Comment by l4v4ph0x** [comment]
yup there very much to change to make it better, flyff.h look terrible now :P

**Comment by DjTrilogic** [comment]
I'm still struggling for the Perin Converter & Collision addresses.
Your help will be much appreciated

**Comment by l4v4ph0x** [comment]
I don't have tut video for it.
but I found it by testing which function sends packet to server to convert penya

**Comment by DjTrilogic** [comment]
I also found this function, but i don't understand how you replay it trough writing in memory....

What about collision stuff ?

**Comment by l4v4ph0x** [comment]
i copyed perin convert function to difference place in mem which hash infinity loop and i change jmp or nop when need to convert perin or not, so it spams sending packets.

collision i found that way by having chance to look how admin panel works and i searched what changes when enabling collision.

**Comment by DjTrilogic** [comment]
Thank’s a lot mate ! I learned a lot of stuff trough your project ! (y)

Provenance : Courrier pour Windows 10

De : lavaphox
Envoyé le :27 September 2017 21:39
À : l4v4ph0x/floral_flyff_cheat
Cc : Abdessamad Benslimane; State change
Objet :Re: [l4v4ph0x/floral_flyff_cheat] Offset retrieving method (#1)

i copyed perin convert function to difference place in mem which hash infinity loop and i change jmp or nop when need to convert perin or not, so it spams sending packets.
collision i found that way by having chance to look how admin panel works and i searched what changes when enabling collision.
